src/libraries/System.Data.Common/src/System/Data/ProviderBase/SchemaMapping.cs

```diff
@@ -1139,7 +1149,17 @@ private void RollbackAddedItems(List<object>? items)
             return dataValues;
         }
 
-        [RequiresUnreferencedCode(DataSet.RequiresUnreferencedCodeMessage)]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
This is only called from a method that is already marked `RequiresUnreferencedCode`. Can we just update the message on `SetupSchemaWithKeyInfo` and say that both:

* chapterValue's type cannot be statically analyzed
* When _loadOption is set, members from types used in the expression column may be trimmed if not referenced directly.

Keep the `RequiresUnreferencedCode` on the LoadOption setter as well. This is just about not needing to refactor this method out here.